scripts/compose.py

```diff
@@ -1211,9 +1240,10 @@ def __init__(self, **options):
         super(AgentRUMJS, self).__init__(**options)
         self.agent_branch = options.get("rum_agent_branch", self.DEFAULT_AGENT_BRANCH)
         self.agent_repo = options.get("rum_agent_repo", self.DEFAULT_AGENT_REPO)
-        self.depends_on = {
-            "apm-server": {"condition": "service_healthy"},
-        }
+        if options.get("enable_apm_server", True):
```

Comment:
is metricbeat ok with the config as-is without an apm-server?  It's possible metricbeat.yml needs some change if no apm-server is going to be available, not certain.

Reply:
if you set `--no-apm-server` means you should have an `apm-server` running in someplace, so you have to have configured `--apm-server-url`

Reply:
I don't generally agree with that but I'm ok with imposing that limitation for metricbeat as this is apm-it after all

Reply:
I still think this may be a problem though: https://github.com/elastic/apm-integration-testing/blob/6ee572b5efbb593204f97427139421727684bb6e/docker/metricbeat/metricbeat.yml#L29

Reply:
it has a similar issue with the `--no-kibana` https://github.com/elastic/apm-integration-testing/blob/6ee572b5efbb593204f97427139421727684bb6e/docker/metricbeat/metricbeat.yml#L7

Reply:
and here https://github.com/elastic/apm-integration-testing/blob/6ee572b5efbb593204f97427139421727684bb6e/docker/apm-server/Dockerfile#L21

Reply:
I've added the proper environment variable with default value on every place it is needed. I've created a unit test for each iteration on {}_elasticsearch_urls and enable_apm_server parameters